app/src/main/res/xml/preferences_playback.xml

```diff
@@ -71,6 +71,11 @@
                 android:key="prefPlaybackSpeedLauncher"
                 android:summary="@string/pref_playback_speed_sum"
                 android:title="@string/pref_playback_speed_title"/>
+        <SwitchPreference
+                android:defaultValue="false"
+                android:key="prefPlaybackTimeRespectsSpeed"
+                android:summary="@string/pref_playback_time_respects_speed_sum"
+                android:title="@string/pref_playback_time_respects_speed_title"/>
```

Comment:
I am not totally happy with having this setting in the category "Buttons for playback control". Do you have an idea where this setting could instead be located? Maybe somewhere on the "User interface" screen?

Reply:
It makes sense to me to have it near the "playback speeds" setting (where you select your preferred speed)
But I can move it under "user interface" / "behavior" if you prefer (I'm not sure we should create a new category)

Let me know what you prefer

![Screenshot_1558817875](https://user-images.githubusercontent.com/365097/58374553-8b1bcc00-7f40-11e9-82c4-51767ee683fb.png)

Reply:
Maybe we could rename the category "Buttons for playback control" to just "Playback control". The time definitely is not a button but I would count it as control element. Then we can leave the setting where it currently is. Could you please update the string and its name if you agree (just the English version)?

Reply:
Please only rename the English version. Otherwise, we might get conflicts when pulling the translations from Transifex.

Reply:
I've just rename the key, not the content.
Do you want me to keep `<string name="buttons` for every other languages but have ` <string name="playback_control"` for english ?

Reply:
Yes, please just keep other languages untouched. The changes will be overridden anyways when pulling updates from Transifex.

Reply:
ok done